File: back/src/PortfolioDev.Domain/Models/Projeto.cs
namespace PortfolioDev.Domain.Models;

public class Projeto
{
	public int Id { get; set; }
	public string Nome { get; set; }
	public string Descricao { get; set; }
	
	public List<Linguagem> Linguagens { get; set; }
	public List<Framework> Frameworks { get; set; }
	public List<Ferramenta> Ferramentas { get; set; }
	
	public int PortfolioId { get; set; }
	public Portfolio Portfolio { get; set; }
}

File: front/PortfolioDev/src/models/projeto/projeto.ts
export class Projeto {
  public id: number | null;
  public nome: string | null;
  public descricao?: string | null;

  constructor(
    id: number | null,
    nome: string | null,
    descricao?: string | null
  ) {
    this.id = id;
    this.nome = nome;
    this.descricao = descricao;
  }
}
